terraso_backend/apps/graphql/schema/projects.py

```diff
@@ -46,12 +47,26 @@ class Meta:
 class ProjectNode(DjangoObjectType):
     id = graphene.ID(source="pk", required=True)
     seen = graphene.Boolean(required=True)
+    soil_settings = graphene.Field(
+        "apps.soil_id.graphql.soil_data.ProjectSoilSettingsNode",
+        required=True,
+        default_value=ProjectSoilSettings(),
+    )
 
     class Meta:
         model = Project
 
         filterset_class = ProjectFilterSet
-        fields = ("name", "privacy", "description", "updated_at", "group", "site_set", "archived")
+        fields = (
+            "name",
+            "privacy",
+            "description",
+            "updated_at",
+            "group",
+            "site_set",
+            "archived",
+            "soil_settings",
+        )
```

Comment:
qustion: should these be in alphabetical order (or some other order)?

Reply:
i'm not sure how we order fields! i usually default to just tacking new fields onto the end and letting the list grow naturally since it makes it easier to identify merge conflicts. i lean away from manual alphabetical ordering since it requires manual maintenance of the alphabetical ordering that can easily become ostensibly alphabetical but actually not,  but if that's generally how we order the fields i can keep to that

this comment did help me notice the `"soil_settings"` in this list is now redundant though so i will fix that!